### PR TITLE
Added trial salting and hashing to password

### PR DESCRIPTION
**Pull request by kevinquayle**
Added in the framework for storing a salted and hashed password as a proof of concept, and it can be altered once we know what the actual access code should be.

### CODE REVIEW

File: src/components/LoginPage.js [hunk 1]
```diff
@@ -7,15 +7,25 @@ import { useNavigate } from 'react-router-dom';
 const LoginPage = ({ onSuccessfulSubmit }) => {
   const [userCode, setUserCode] = useState('')
   const [agreed, setAgreed] = useState(false)
-  const secretCode = "test";
+  // const secretCode = 'test';
```
Comment (masakiasanuma) [inline src/components/LoginPage.js:10]: I think you can just remove this line so the password can't be seen in plain text

**Review by masakiasanuma — APPROVED** [review]
Everything else looks good! Thanks for adding this, it'll be good enough for this semester in terms of protection.

**Comment by kevinquayle** [comment]
I removed 'test' and the new code is 'h0me care!'

**Review by valistelea — APPROVED** [review]
Great work!

**Pull request merged**